registration-processor/post-processor/registration-processor-printing-stage/src/main/java/io/mosip/registration/processor/print/stage/PrintingStage.java

```diff
@@ -345,6 +343,7 @@ public MessageDTO process(MessageDTO object) {
 
 	private CredentialRequestDto getCredentialRequestDto(String regId) {
```

Comment:
We need to support 2 credential request based on configuration, 1 for print and 1 for digital card service

Reply:
taken care